src/HashFunction/PasswordHasher.php

```diff
@@ -66,7 +66,7 @@ public function compare(string $password, string $hash): bool
     /**
      * @throws ErrorException Error converted to an exception.
      */
-    private static function errorHandler(int $severity, string $message, string $filename, int $line): void
+    private static function errorHandler(int $severity, string $message, string $filename, int $line): bool
```

Comment:
`set_error_handler` expects a closure that can return `false`, though this is a technicality

Reply:
Good catch!